tests/Functional/WebTestCase.php

```diff
@@ -70,4 +72,14 @@ protected static function getVarDir()
     {
         return \substr(\strrchr(static::class, '\\'), 1);
     }
+
+    // To be removed when dropping support of Symfony < 5.3 and use "self::getContainer()" instead.
```

Comment:
Feel free to bump to Symfony 5.4 and remove this BC layer.

Reply:
You mean dropping support of Symfony 4.4?

Reply:
Oops, I forgot we still support 4.4 :)